### ISSUE TITLE
Question on AttSampler

### ISSUE THREAD

**Issue by AlexHu123** [wkcn/AttentionSampler#2]
Hi, will you release the PyTorch version? And can your op support the backpropogation in the loss update ?

**Comment by wkcn** [comment]
Sorry that I will not release the PyTorch version, since I'm busy now.

**Comment by wkcn** [comment]
The PyTorch version has been released.
https://github.com/wkcn/AttentionSampler

**Comment by AlexHu123** [comment]
Thanks for your kind support !
